diagnostic_msgs/KeyValue[]   configs
